src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformAppCompatTextView.java

```diff
@@ -0,0 +1,44 @@
+package com.microsoft.maui;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.widget.AppCompatTextView;
+
+public class PlatformAppCompatTextView extends AppCompatTextView {
+    public PlatformAppCompatTextView(@NonNull Context context) {
+        super(context);
+    }
+
+    private boolean isFormatted;
+
+    /**
+     * Sets isFormatted=true, used for FormattedString content
+     * @param text
+     * @param type
+     */
+    @Override
+    public void setText(CharSequence text, BufferType type) {
+        isFormatted = !(text instanceof String);
+        super.setText(text, type);
+    }
```

Comment:
Note that calling `TextView.Text = "foo"` will call inside here:

* https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/android/widget/TextView.java;l=7077

The C# `TextView.Text` property is bound to `setText(CharSequence)`.